### PR TITLE
testing/eolie: new aport

### PR DESCRIPTION
**Pull request by drebrez**

### CODE REVIEW

File: testing/eolie/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+pkgname=eolie
+pkgver=0.9.63
+pkgrel=0
+pkgdesc="Web browser for GNOME"
+url="https://gitlab.gnome.org/World/eolie"
+arch="all"
+license="GPL-3.0-or-later"
+depends="webkit2gtk gtkspell3 py3-cairo py3-gobject3"
+makedepends="meson gobject-introspection-dev webkit2gtk-dev python3-dev py3-gobject3-dev"
+source="$pkgname-$pkgver.tar.xz::https://gitlab.gnome.org/World/eolie/uploads/d42b466752729a0d3fa828a721e25043/eolie-0.9.63.tar.xz
```
Comment (Cogitri) [inline testing/eolie/APKBUILD]: Please add `desktop-file-utils` and `appstream-glib` as ` checkdepends`, they're used for additional tests.

```
...
Message: Compiling resources
Configuring AboutDialog.ui using configuration
Found pkg-config: /usr/bin/pkg-config (1.6.3)
Program desktop-file-validate found: NO
Program appstream-util found: NO
Program glib-compile-schemas found: YES (/usr/bin/glib-compile-schemas)
Message: Making a list of icons
...
```

**Comment by Cogitri** [comment]
Oh, also:

```
>>> WARNING: eolie*: Found /usr/share/locale
```

Please add a $pkgname-lang subpackage

File: testing/eolie/APKBUILD (hunk 1 above)
Comment (drebrez) [reply testing/eolie/APKBUILD]: Added 👍

**Comment by drebrez** [comment]
> Please add a $pkgname-lang subpackage

Added 👍

**Comment by Cogitri** [comment]
Thanks for your contribution and for the quick responses :)

**Pull request closed without merging**